learning/tour-of-beam/frontend/lib/pages/welcome/screen.dart

```diff
@@ -201,12 +204,14 @@ class _Buttons extends StatelessWidget {
 }
 
 class _SdkButton extends StatelessWidget {
-  final String sdk;
-  final String selectedSdk;
+  final String value;
+  final String groupValue;
+  final void Function()? onChanged;
```

Comment:
`final ValueChanged<String> onChanged;`

Reply:
The argument type 'void Function(String)' can't be assigned to the parameter type 'void Function()?'.

Also, I expected the value to be changed with bloc. Are you sure we need an onChanged function with a parameter?

I think a function without parameters and with a name like "changeValue" might fit better here.

Reply:
I'm also not sure about the value and groupValue naming: they presume that it's a general widget, but ours is a specific widget called SdkButton.